### PR TITLE
Add delegation example diagrams

### PR DESCRIPTION
**Pull request by dplumb94**
- Adds a link to the delegation example from the feature page for more high-level info
- Replaces the condensed example with the full example from the RFC
- Adds diagrams for each step of the example

### CODE REVIEW

File: docs/0.2/using_pike.md [hunk 1]
```diff
@@ -384,86 +385,168 @@ Alpha Company also needs to define a role for drivers:
...
+    - beta
   inherit_from: []
+  active: true
+```
+
+![]({% link docs/0.2/images/pike_diagrams/delta.svg %})
+
+To allow its agents to start operating and inspecting the Delta tanks, the Beta
+Company admin updates the `beta.Drivers` role to also include the
+`tankops::can-decommission` permission and add `delta.TankOperator` as a super role.
```
Comment (dnewh) [inline docs/0.2/using_pike.md:479]: This line should wrap at 80 characters

File: docs/0.2/using_pike.md [hunk 2]
```diff
@@ -337,42 +341,39 @@ The data model for the t-shirt tank may look something like this:
 The smart contract for operating tanks (named "tankops") would also define
 several permissions:
 
-- `tankops::can-drive` allows agents to submit a transaction to update the
-`is_driving` boolean.
+- `tankops::can-drive` allows agents to submit a transaction to update the `is_driving`
```
Comment (dnewh) [inline docs/0.2/using_pike.md:344]: This line should wrap at 80 characters

**Review by dnewh — CHANGES_REQUESTED** [review]

File: docs/0.2/using_pike.md (hunk 1 above)
Comment (dplumb94) [reply docs/0.2/using_pike.md]: fixed

File: docs/0.2/using_pike.md (hunk 2 above)
Comment (dplumb94) [reply docs/0.2/using_pike.md]: fixed

**Review by dnewh — APPROVED** [review]

**Review by agunde406 — APPROVED** [review]

**Pull request merged**